### PR TITLE
Add 6 speakers

### PR DESCRIPTION
**Pull request by lincolnkoh99**

### CODE REVIEW

File: src/data/speakers.js [hunk 1]
```diff
@@ -134,6 +134,42 @@ const GUEST_SPEAKERS = [
...
   },
+  {
+    name: 'Prateek Saxena',
+    role: 'Associate Professor',
+    company: 'School of Computing, NUS',
+    image: '/speakers/SOC NUS - Prateek Saxena.png',
+  },
+  {
+    name: 'Aashish Kolluri',
+    role: 'PhD student',
```
Comment (jerrenss) [inline src/data/speakers.js:145]: Can caps the S in Student thanks! For all below too

**Review by jerrenss — COMMENTED** [review]
Just a small fix, other than that looks good!

File: src/data/speakers.js [hunk 2]
```diff
@@ -134,6 +134,42 @@ const GUEST_SPEAKERS = [
     company: 'SeaMoney',
     image: '/speakers/SeaMoney - E Teng.jpg',
   },
+  {
+    name: 'Prateek Saxena',
+    role: 'Associate Professor',
+    company: 'School of Computing, NUS',
```
Comment (jerrenss) [inline src/data/speakers.js:140]: Can use 'NUS School of Computing' instead

**Comment by lincolnkoh99** [comment]
I just made these minor changes

**Comment by jerrenss** [comment]
Thanks!

**Pull request merged**